### ISSUE TITLE
Schematic Submission

### ISSUE THREAD

**Issue by timChase98** [fviveirosraposo/Bom-Submission-#2]
- [x] Review the submission guidelines [HERE](https://github.com/timChase98/388Example)
- [x] All project files should be version controlled
- [x] Use power flags instead of named nets for power distribution 
- [x] All nets should be at right angles
- [x] How does power come onto your board? 
- [x] Rotate J2 90 degrees 
- [x] Add a 100nF cap across the power pins of the 328pb
- [x] Add a pull up resistor to the reset line of the uController

Please address these issues and push your changes. Then create a new release, increment the tag and close this issue. I will review the changes and mark this as complete or reopen the issue with additional feedback.

**Comment by fviveirosraposo** [comment]
@timChase98 
Hey Tim, I think I have addressed all issues. The power comes into the board via the USB connector or the 5v voltage regulator and will be using a 9v battery for that. Let me know if there are any more issues.   

Thank you

**Comment by timChase98** [comment]
- [x] Address the above issues
- [x] Check the datasheet for the 328pb for reset circuitry 
- [x] Check the datasheet for the LM1086 applications
- [x] Use power flags instead of named nets for power distribution

**Comment by fviveirosraposo** [comment]
@timChase98 
I think I have all issues addressed now, I have an ERC error when I added one of the power flags that's why I didn't have it before, but I included it now. I am actually using an lm7805, I had the wrong footprint in, so I fixed that and according to its datasheet my circuit is good. I also fixed the reset circuit and put the resistor to 5v instead of ground.  I also put version 2.0 in all my files since I never add any version before I just started at 2.0. I also updated my board according to the fixes made so if there is no more errors in the schematic the board should be good.
